src/python/WMCore/JobSplitting/EventAwareLumiByWork.py

```diff
@@ -258,6 +259,8 @@ def lumiListFromACDC(couchURL=None, couchDB=None, filesetName=None, collectionNa
         This is not implemented yet
         :return:
         """
+        from WMCore.ACDC.DataCollectionService import DataCollectionService
```

Comment:
Can this go to the top?

Reply:
It was in the top but I moved it over here because - I think - it would break CRAB.